MergerService/src/run.cs

```diff
@@ -0,0 +1,166 @@
+using MergerLogic.Batching;
+using MergerLogic.DataTypes;
+using MergerLogic.ImageProccessing;
+using MergerLogic.Utils;
+using MergerService.Controllers;
+using System.Diagnostics;
+
+namespace MergerService.Src
+{
+    public class Run
+    {
+        private static List<Data> BuildDataList(Source[] paths, int batchSize)
+        {
+            List<Data> sources = new List<Data>();
+
+            if (paths.Length != 0)
+            {
+                sources.Add(Data.CreateDatasource(paths[0].Type, paths[0].Path, batchSize, true));
```

Comment:
wont work with http/none default grid/ none default origin

Reply:
http support isn't critical and requires making the extent optional

Reply:
Will add support for HTTP in next PR.